react/src/common/utils.ts

```diff
@@ -105,10 +100,14 @@ export const createLink = ({ type, id, moduleId }: CreateLink) => {
  */
 
 export const setQueryStringWithoutPageReload = (qsValue: string) => {
-  const newurl = window.location.protocol + '//' +
-                 window.location.host +
-                 window.location.pathname +
-                 qsValue;
+  const newurl =
+    window.location.protocol +
+    '//' +
+    window.location.host +
+    window.location.pathname +
+    qsValue;
 
   window.history.pushState({ path: newurl }, '', newurl);
-};
\ No newline at end of file
+};
+
+export const IS_DEV = process.env.NODE_ENV === 'development';
```

Comment:
this is the only new thing in this file. The rest were automatic styling updates